pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
> Afaik this isn't "really" part of the public API due to the contents of pluggy.__all__?

Oh that's a good point.

[tox doesn't use it as well](https://github.com/tox-dev/tox/search?utf8=%E2%9C%93&q=varnames), neither does [devpi](https://bitbucket.org/hpk42/devpi) using a local search (BB doesn't seem to have a search feature).

I think we can safely rename it to an internal function to avoid confusion then. IMHO no need to rename it since nobody outside pluggy uses it as far as we know, and in case someone complains and makes a good case for it we can reintroduce it and promote to the public API.

`0.X` is the period of time we can break things if we think that's better. 😉